scripts/fetch/view_limits.R

```diff
@@ -0,0 +1,90 @@
+#' @title Fetch viz view limits
+#' @param viz a vizlab object including a \code{parameter_spatial} parameter input.
+#' @details \code{parameter_spatial} must include:
+#' /describe{
+#'   \item{id}{parameter_spatial}
+#'   \item{crs}{valid crs for \pkg{sf}}
```

Comment:
`crs` appears to be hard coded now. are we good with that?

Reply:
That is the CRS for lat/long - 

This function assumes the bbox that you give it in params is a lat/lon one. I am OK with that, but we should probably be explicit in the documentation and might want to check that the values are within the bounds we expect for lat/lon.